lib/widgets/layouts/adaptive_layout/adaptive_scaffold_view.dart

```diff
@@ -174,7 +178,7 @@ class AppScaffoldView extends StatelessWidget {
         return 0;
       case AdaptiveDestinationEnum.rooms:
         return 1;
-      case AdaptiveDestinationEnum.stories:
+      case AdaptiveDestinationEnum.settings:
```

Comment:
I think you should create extension on this Enum
then refactor this function and also `onDestinationSelected`